### ISSUE TITLE
Add to OH2 Database: Linear PS15Z-2 Appliance Module

### ISSUE THREAD

**Issue by mhilbush** [cdjackson/HABmin2#166]
I'm a new user of OH2, so still getting my feet wet.  I hope this is sufficient to get this into the database.

Brand Name: Linear
Product Name: PS15Z-2
Description: Z-Wave Plug-in Appliance Module

Manufacturer ID: 014f
Product Type: 5250
Product Id: 3030
Version: 5.41

Command Class: 20
Command Class: 25
Command Class: 86
Command Class: 27
Command Class: 2b
Command Class: 2c
Command Class: 70
Command Class: 72
Command Class: 73
Command Class: 75

Config Param: 104
Name: Flash Mode
Description: Set parameter to 1 for normal ON/OFF operation, set parameter to 0 to enable flash mode

[linear_ps15z2_instruction_manual.pdf](https://github.com/cdjackson/HABmin2/files/257295/linear_ps15z2_instruction_manual.pdf)

**Comment by cdjackson** [comment]
Ideally I’d like to get an XML file for this device - does OH generate one - it would be in the /userdata/zwave folder?

**Comment by mhilbush** [comment]
Sure enough.  I guess that would be why I'm on the steep part of the learning curve...  :-)

[zwave_linear_ps15z-2.txt](https://github.com/cdjackson/HABmin2/files/257663/zwave_linear_ps15z-2.txt)

**Comment by cdjackson** [comment]
Thanks.

http://www.cd-jackson.com/index.php/zwave/zwave-device-database/zwave-device-list/devicesummary/352

**Comment by cdjackson** [comment]
https://github.com/openhab/openhab2-addons/pull/912 now merged

**Comment by cdjackson** [comment]
Please advise if this can be closed.

**Comment by mhilbush** [comment]
I'm sorry.  I want to test this, but I haven't figured out how to get a new build to place in my environment.

**Comment by cdjackson** [comment]
In paper ui, uninstall the binding and re-install it again.

**Comment by mhilbush** [comment]
I'm sorry for the follow on questions...  Does this assume that I've installed the online distro of OH2?  If so, I've installed the offline distro, and will need to reinstall the online distro?

**Comment by cdjackson** [comment]
Yes - that's for the online distro. If you install the offline, you need to reinstall the whole system.

**Comment by mhilbush** [comment]
Argh...  So it would probably be best to close this while I take care of that.  I can always reopen if there's an issue?

**Comment by mhilbush** [comment]
I rebuilt my OH2 install from scratch using the online distro -- should've done this the first time, but I didn't know any better...  I also reset the zwave network, so there was only the controller and Linear PS15Z-2 device in play.

I included the PS15Z-2, and it was discovered correctly.  I did the include using my Aeon Gen5 stick in the "manual" mode (i.e. it was not connected to the computer).  After connecting the stick to the USB port, the device was discovered and the PS15Z-2 was correctly assigned to the node (node 2).

I did see an exception in the debug log file.  I'm not sure of the cause, so I'm reopening this issue, and including the log file entries from the point OH2 was started until the node 2 processing was complete.  The exception occurred at 13:09:19.420.

[node-2-exception.txt](https://github.com/cdjackson/HABmin2/files/261941/node-2-exception.txt)
